### PR TITLE
chore: remove icon-toggle

### PR DESCRIPTION
**Pull request by moog16**
related #4225

### CODE REVIEW

File: demos/theme/_theme-shared.scss [hunk 1]
```diff
@@ -345,12 +345,12 @@ figure {
   display: inline-flex;
 }
 
-.demo-icon-toggle-row {
+.demo-icon-button-row {
```
Comment (kfranqueiro) [inline demos/theme/_theme-shared.scss]: Also rename comment above this section to Icon Button demo

File: demos/theme/index.html [hunk 2]
```diff
@@ -463,14 +463,14 @@ <h3 class="mdc-typography--headline5 demo-component-section__heading">
       </section>
 
       <section class="demo-component-section">
-        <i id="icon-toggle" class="demo-anchor-with-toolbar-offset"></i>
+        <i id="icon-button--toggle" class="demo-anchor-with-toolbar-offset"></i>
         <h3 class="mdc-typography--headline5 demo-component-section__heading">
           Icon Toggle
-          <a href="#icon-toggle" class="demo-component-section__permalink" title="Permalink to the theme demo for the icon toggle component">#</a>
+          <a href="#icon-button--toggle" class="demo-component-section__permalink" title="Permalink to the theme demo for the icon toggle component">#</a>
```
Comment (kfranqueiro) [inline demos/theme/index.html]: icon toggle -> icon button

File: packages/material-components-web/index.js [hunk 3]
```diff
@@ -32,7 +32,6 @@ import * as floatingLabel from '@material/floating-label/index';
 import * as formField from '@material/form-field/index.ts';
 import * as gridList from '@material/grid-list/index';
 import * as iconButton from '@material/icon-button/index.ts';
-import * as iconToggle from '@material/icon-toggle/index';
```
Comment (kfranqueiro) [inline packages/material-components-web/index.js]: You forgot to remove `iconToggle` from exports below (shouldn't this break?)

Comment (moog16) [reply packages/material-components-web/index.js]: ah that probably came up in npm run build but not the dev server. Thanks!

**Pull request merged**